ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java

```diff
@@ -21,6 +21,8 @@
     private final static By MANAGE_VISIT_TYPES = By.cssSelector("#legacyui-manageVisitTypes a");
     private static final By MANAGE_PROVIDERS = By.cssSelector("#content a[href='/openmrs/admin/provider/index.htm']");
     private static final By MANAGE_PERSONS = By.cssSelector("#content a[href=\"/openmrs/admin/person/index.htm\"]");
+    private static final By MANAGE_HTMLFORMS_PAGE = By.cssSelector("#content a[href*=\'/module/htmlformentry/htmlForms.list\']");
```

Comment:
no need to escape ' and replace all \" with '

Reply:
Resolved thanks